app/models/pages/WhatIsNext.scala

```diff
@@ -26,16 +26,21 @@ object WhatIsNext extends Enumerable.Implicits {
   case object DeclareTheTrustIsUpToDate extends WithName("declare") with WhatIsNext
   case object MakeChanges extends WithName("make-changes") with WhatIsNext
   case object CloseTrust extends WithName("close-trust") with WhatIsNext
+  case object NoLongerTaxable extends WithName("no-longer-taxable") with WhatIsNext
   case object GeneratePdf extends WithName("generate-pdf") with WhatIsNext
 
   val values: List[WhatIsNext] = List(
-    DeclareTheTrustIsUpToDate, MakeChanges, CloseTrust, GeneratePdf
+    DeclareTheTrustIsUpToDate, MakeChanges, CloseTrust, NoLongerTaxable, GeneratePdf
   )
 
-  def options(is5mldEnabled: Boolean): List[(RadioOption, String)] =
+  def options(is5mldEnabled: Boolean, isTrust5mldTaxable: Boolean): List[(RadioOption, String)] = {
+    val suffix: String = if (is5mldEnabled){"5mld"}else{""}
     values
       .filterNot(_ == GeneratePdf && !is5mldEnabled)
-      .map(value => (RadioOption("declarationWhatNext", value.toString), s"declarationWhatNext.$value.hint"))
+      .filterNot(_ == NoLongerTaxable && !is5mldEnabled)
+      .filterNot(_ == NoLongerTaxable && !isTrust5mldTaxable)
```

Comment:
Can this be `.filterNot(_ == NoLongerTaxable && (!is5mldEnabled || !isTrust5mldTaxable))`

Reply:
Which can then become `.filterNot(_ == NoLongerTaxable && !(is5mldEnabled && isTrust5mldTaxable))`

Reply:
I could be wrong, but i think that would allow it through if we were in 4MLD mode but it was dealing with a 5mld taxable registration?

It's not an issue we would ever hit in prod but it would mess up some local/staging testing when using stubbed data